homeassistant/components/forked_daapd/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Owntone",
   "documentation": "https://www.home-assistant.io/integrations/forked_daapd",
   "codeowners": ["@uvjustin"],
-  "requirements": ["pyforked-daapd==0.1.11", "pylibrespot-java==0.1.0"],
+  "requirements": ["pyforked-daapd==0.1.14", "pylibrespot-java==0.1.1"],
```

Comment:
Please publish github releases corresponding to the PyPI releases to make it easier for the community to track changes to the library.

I recommend automating this with a github action.

Reply:
How about just tags instead of releases?

Reply:
Yes, that's ok. The important thing is to be able to easily identify the commit on github that corresponds to the release on PyPI and be able to use the github compare feature.